src/main/java/com/book/booksstore/model/Book.java

```diff
@@ -0,0 +1,32 @@
+package com.book.booksstore.model;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.UniqueConstraint;
+import java.math.BigDecimal;
+import lombok.Getter;
+import org.antlr.v4.runtime.misc.NotNull;
+
+@Entity
+@Getter
+public class Book {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @NotNull
+    public String title;
```

Comment:
```suggestion
    @Column(nullable = false) 
    public String title;
```
the same for the other required fields

Reply:
done
![image](https://github.com/user-attachments/assets/6f5e7a6a-5bfc-4b2d-ae63-4728ff252f08)